commercetools-models/src/test/java/io/sphere/sdk/carts/commands/CartCreateCommandIntegrationTest.java

```diff
@@ -231,6 +231,18 @@ public void createCartWithStoreReference() {
         });
     }
     
+    @Test
+    public void createCartInStore(){
+        StoreFixtures.withStore(client(), store -> {
```

Comment:
Here it would be nice to use a static import - that would make it more consistent with the other code 😉